spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -723,7 +727,6 @@ public DeleteGranularity deleteGranularity() {
   }
 
   public boolean useDVs() {
```

Comment:
It also feels like we should really just remove this method and have it embedded in deleteFileFormat().

Reply:
Changed my mind on second thought, see https://github.com/apache/iceberg/pull/11986#discussion_r1919571061 for my rationale